src/meta/proto-conv/tests/it/v122_virtual_schema.rs

```diff
@@ -0,0 +1,222 @@
+                ce::TableField::new("int32", ce::TableDataType::Number(NumberDataType::Int32)),
+                ce::TableField::new("int64", ce::TableDataType::Number(NumberDataType::Int64)),
+                ce::TableField::new("uint8", ce::TableDataType::Number(NumberDataType::UInt8)),
+                ce::TableField::new("uint16", ce::TableDataType::Number(NumberDataType::UInt16)),
+                ce::TableField::new("uint32", ce::TableDataType::Number(NumberDataType::UInt32)),
+                ce::TableField::new("uint64", ce::TableDataType::Number(NumberDataType::UInt64)),
+                ce::TableField::new(
+                    "float32",
+                    ce::TableDataType::Number(NumberDataType::Float32),
+                ),
+                ce::TableField::new(
+                    "float64",
+                    ce::TableDataType::Number(NumberDataType::Float64),
+                ),
+                ce::TableField::new("date", ce::TableDataType::Date),
+                ce::TableField::new("timestamp", ce::TableDataType::Timestamp),
+                ce::TableField::new("string", ce::TableDataType::String),
+                ce::TableField::new("struct", ce::TableDataType::Tuple {
+                    fields_name: vec![s("foo"), s("bar")],
+                    fields_type: vec![ce::TableDataType::Boolean, ce::TableDataType::String],
+                }),
+                ce::TableField::new(
+                    "array",
+                    ce::TableDataType::Array(Box::new(ce::TableDataType::Boolean)),
+                ),
+                ce::TableField::new("variant", ce::TableDataType::Variant),
+                ce::TableField::new("variant_array", ce::TableDataType::Variant),
+                ce::TableField::new("variant_object", ce::TableDataType::Variant),
+                // NOTE: It is safe to convert Interval to NULL, because `Interval` is never really used.
+                ce::TableField::new("interval", ce::TableDataType::Null),
+                ce::TableField::new("bitmap", ce::TableDataType::Bitmap),
+                ce::TableField::new("geom", ce::TableDataType::Geometry),
+            ],
+            btreemap! {s("a") => s("b")},
+        )),
+        engine: "44".to_string(),
+        storage_params: None,
+        part_prefix: "".to_string(),
+        engine_options: btreemap! {s("abc") => s("def")},
+        options: btreemap! {s("xyz") => s("foo")},
+        cluster_key: Some("(a + 2, b)".to_string()),
+        cluster_key_seq: 0,
+        created_on: Utc.with_ymd_and_hms(2014, 11, 28, 12, 0, 9).unwrap(),
+        updated_on: Utc.with_ymd_and_hms(2014, 11, 29, 12, 0, 10).unwrap(),
+        comment: s("table_comment"),
+        field_comments: vec!["c".to_string(); 21],
+        virtual_schema: Some(ce::VirtualDataSchema {
+            fields: vec![ce::VirtualDataField {
+                name: "field_0".to_string(),
+                data_types: vec![
+                    ce::VariantType::Jsonb,
```

Comment:
Usually we don't have so many types for a virtual column, I think we can just write two or three types for test data